app/src/test/java/com/example/nvdovin/weatherapp/domain/utils/sharedpreferences/SharedPrefsTest.java

```diff
@@ -0,0 +1,56 @@
+package com.example.nvdovin.weatherapp.domain.utils.sharedpreferences;
+
+import android.content.SharedPreferences;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(org.mockito.junit.MockitoJUnitRunner.class)
+public class SharedPrefsTest {
+    private final static String LAST_UPDATE_TIME_KEY = "LAST_UPDATE_TIME_KEY";
+    @Mock
+    SharedPreferences sharedPreferences;
+    @Mock
+    SharedPreferences.Editor editor;
+    private SharedPrefs sharedPrefs;
+
+    @Before
+    public void setUp() throws Exception {
+        sharedPrefs = new SharedPrefs(sharedPreferences);
+    }
+
+
+    @Test
+    public void lastUpdateExceededLimit() throws Exception {
+
+        when(sharedPreferences.getLong(LAST_UPDATE_TIME_KEY, 0))
+                .thenReturn(System.currentTimeMillis());
+
+        assertFalse(sharedPrefs.lastUpdateExceededLimit());
+
+        when(sharedPreferences.getLong(LAST_UPDATE_TIME_KEY, 0))
+                .thenReturn(0L);
+
+        assertTrue(sharedPrefs.lastUpdateExceededLimit());
+    }
+
+    @Test
+    public void getTempScaleFromPrefs() throws Exception {
+
+        when(sharedPreferences.getInt(anyString(), anyInt())).thenReturn((int) Math.random());
```

Comment:
It will make more sense to return a value, and then to assert that this value was returned by tested method getTempScaleFromPrefs().